barcodes_generator_product/models/product_template.py

```diff
@@ -27,7 +25,6 @@ class ProductTemplate(models.Model):
 
     generate_type = fields.Selection(
         string="Generate Type",
-        selection=_GENERATE_TYPE,
```

Comment:
Why did you removed this line ?

Reply:
because it's a related field
does not make sense re-defined selection attribute

Reply:
see https://github.com/odoo/odoo/blob/14.0/odoo/fields.py#L2250-L2251

Reply:
Thanks a lot ! I didn't know.